cvprac/cvp_api.py

```diff
@@ -3828,11 +3781,9 @@ def device_decommissioning(self, device_id, request_id):
                 url = '/api/resources/inventory/v1/DeviceDecommissioningConfig'
                 self.log.debug('v7 ' + str(url) + ' ' + str(payload))
                 return self.clnt.post(url, data=payload, timeout=self.request_timeout)
-        else:
-            self.log.warning(
-                'Device with %s serial number does not exist (or is not registered) to decommission'
-                % device_id)
-            return None
+        self.log.warning(f"Device with {device_id} serial number does not exist"
```

Comment:
would we want this warning to be logged even when outer if was met (device exists) but inner wasn't (version compare evaluates to false)?
in earlier logic we would not have but in newer logic we will since we removed the else

not sure if such a flow would happen (lack the context) but putting it out there 👍🏻

Reply:
Added back else and left return out of else.